spec/index.html

```diff
@@ -293,13 +294,16 @@ <h3>RDF Vocabularies and Namespace IRIs</h3>
       </tr>
     </table>
 
-    <p>In some serialization formats it is common to abbreviate <a>IRIs</a>
-      that start with <a>namespace IRIs</a> by using a
-      <a>namespace prefix</a> in order to assist readability. For example, the IRI
-      <code>http://www.w3.org/1999/02/22-rdf-syntax-ns#XMLLiteral</code>
+    <p>In some serialization formats, it is common
+      to associate some [=namespace IRIs=] with arbitrary [=namespace prefixes=],
+      and to improve readability by abbreviating <a>IRIs</a> that start with one of those <a>namespace IRIs</a>
+      by using the corresponding <a>namespace prefix</a>.
+      For example,
+      based on the prefix mapping in <a href="#tab-vocab-ns"></a>,
+      the IRI <code>http://www.w3.org/1999/02/22-rdf-syntax-ns#XMLLiteral</code>
       would be abbreviated as <code>rdf:XMLLiteral</code>.
-      Note however that these abbreviations are <em>not</em> valid IRIs,
-      and must not be used in contexts where IRIs are expected.
+      Note however that these abbreviations are <em>not</em> valid IRIs (or only coincidentally),
+      and must not be used in syntactic contexts where IRIs are expected.
       Namespace IRIs and namespace prefixes are <em>not</em> a formal part of the
       RDF data model. They are merely a syntactic convenience for
       abbreviating IRIs.</p>
```

Comment:
Try this on for size...
```suggestion
      Note, however, that these "prefixed name" abbreviations are generally <em>not</em> valid IRIs,
      and must not be used in contexts where only non-prefixed IRIs are expected.
      Namespace IRIs and namespace prefixes are <em>not</em> a formal part of the
      RDF data model. They are merely a convenience for
      abbreviating IRIs in some syntaxes.</p>
```

Reply:
I like that one as well. @pchampin, I think we can take this one instead of the one that [I had proposed](https://github.com/w3c/rdf-concepts/pull/194#discussion_r2060023057).